lib/suspenders/app_builder.rb

```diff
@@ -54,6 +54,10 @@ def configure_newrelic
       template 'newrelic.yml.erb', 'config/newrelic.yml'
     end
 
+    def configure_honeybadger
+      copy_file 'honeybadger.rb', 'config/initializers/honeybadger.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.